apps/core/templates/core/home-authenticated.html

```diff
@@ -3,32 +3,214 @@
+  <div class="grid-x">
+    <div class="cell large-6">
+      <div id="pixels_by_OmicsArea" style="height: 400px"></div>
+    </div>
+    <div class="cell large-6" style="margin: auto;">
+      <!-- coming soon -->
+    </div>
+  </div>
+
+  <h4 style="text-align: center;"> ... characterized by tags. </h4>
   <div class="grid-x">
-    <div class="cell">
-      <div id="pixels-by-species" style="height: 500px"></div>
+    <div class="cell large-6">
+        <div id="tags_charts" style="height: 400px;  "></div>
     </div>
+    <div class="cell large-6" style="margin: auto;">
+      <!-- coming soon -->
+    </div>
+  </div>
+
+  <h4 style="text-align: center;"> New data added over time... </h4>
+  <div class="grid-x">
+    <div class="cell large-12" style="margin: auto;">
+      <!-- coming soon -->
+    </div>
+    <div class="cell large-12">
+      <div id = "CD_div" style="width:100%;">
+        <div id="calendar_basic" style="height: 200px; width: 600px"></div>
+      </div>
+
+    </div>
+  </div>
+
+<h4 style="text-align: center;"> ...by a community of experts. </h4>
+  <div class="grid-x">
+    <div class="cell large-6" style="margin: auto;">
+      <!-- coming soon -->
+    </div>
+    <div class="cell large-6">
+        <table style="margin: 0 auto;">
+          <thead>
+            <tr>
+              <th width="200">Names</th>
+              <th>Email</th>
+              <th width="150">Date joined</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for first_name,last_name, email, date in pixelers_infos.data %}
+            <tr>
+              <td>{{first_name}} {{last_name}}</td>
+              <td>{{email}}</td>
+              <td>{{date}}</td>
+            </tr>
+            {% endfor %}
+          </tbody>
+        </table>
+    </div>
+
   </div>
 {% endblock content %}
 
 {% block javascript %}
   <script type="text/javascript" src="//www.gstatic.com/charts/loader.js"></script>
   <script type="text/javascript">
-    google.charts.load('current', {'packages':['corechart']});
+
+    document.getElementById('CD_div').style.paddingLeft = ((parseInt(document.getElementById('CD_div').offsetWidth)-600)/2)+'px';
+    document.getElementById('CD_div').style.paddingRight = ((parseInt(document.getElementById('CD_div').offsetWidth)-600)/2)+'px';
+
+    document.getElementById('main').style.paddingLeft = '30px';
+    document.getElementById('main').style.paddingRight = '30px';
+
+    google.charts.load('current', {'packages':['corechart', 'bar', 'calendar']});
     google.charts.setOnLoadCallback(drawCharts);
+    google.charts.setOnLoadCallback(drawBasic);
 
     function drawCharts() {
+
+      /* Graphic 1 : Pixels by Species */
       var pixelBySpecies = new google.visualization.PieChart(
         document.getElementById('pixels-by-species')
       );
-      pixelBySpecies.draw(
+
+      var datable_pixels_by_species = new google.visualization.arrayToDataTable(
```

Comment:
- in JS, we use camelCase
- in Python, we use snake_case
- in CSS, we use kebab-case

I am sorry for that, it is not very consistent but it is what we've done in this project because that's what most people do.